### DISCUSSION TITLE
Aligning modules

### DISCUSSION THREAD

**Discussion by AntonioSco** [templaza/astroid-framework#915] [Q&A]
Hi,
I'm trying to do something apparently very simple, which is to use the same header, **Top-a,** to contain two modules: the one containing the e-mail address and a social icon. aligned one after the other. However, even though I indicate the Top-a header for both modules, they end up positioning themselves in two different lines (within the same module I reckon), such as you can see [here](https://permacultura-es.org/Joomla4/).
What is the "trick" to make them appear on the same line and with a minimal header width?

**Comment by Banuk1616** [comment]
Your Layout Manager should look like this.

<img width="1107" alt="header_section" src="https://github.com/user-attachments/assets/2859b45a-9f5f-420d-9f29-b37d1e5366e4" />

You can add the data for contact and social media via the following points. You do not need to create any extra modules for this.

<img width="1203" alt="contact_ssocial" src="https://github.com/user-attachments/assets/c6613f7e-1120-43b9-86bf-990a91629c14" />

Top A in this case is for the contact information and Top B for the social media links.
If you position two modules in Top A, they will be stacked if you don't add extra bootstrap classes, as far as I know.

**Reply by AntonioSco** [reply]
Thanks @Banuk1616 
I initially proceeded like this, but for the e-mail I prefer to have an image instead than a link so I had to create a module for this, also I wanted the image and the social icon close to each other on just one side of the screen, and if I put them in two different headers (Top-a and Top-b) they will remain far apart from each other ... so it is more difficult that I thought apparently.
